build/bin/vlan-filtering

```diff
@@ -0,0 +1,16 @@
+#!/bin/bash -e
+
+vlan_range=1-4094
+
+for bridge in $@; do
+   ip link set dev $bridge type bridge vlan_filtering 1
+   bridge vlan add dev $bridge vid $vlan_range self
+
+   IFS=$'\n'
+   for port in $(bridge link show $bridge | grep -v @ | grep "state forwarding"); do
```

Comment:
not sure if it helps and if it is available on RHEL, but you can use -json to get structured output

Reply:
Thanks! it's working bridge -j vlan show, that makes e2e easier !!!